### ISSUE TITLE
Arbitrarily stopped working

### ISSUE THREAD

**Issue by SolidplasmaStudios** [davidborzek/streamdeck-foobar2000#14]
Hi, so I used to use this all the time, but a few months ago, this streamdeck app just stopped working for me. I can install it fine, but I get the yellow warning icon on my streamdeck whenever I press a button. beefweeb plugin is installed and works, and I can use it in browser. The other strange thing is that the buttons seems to be able to display data from foobar, but not send data, like play/pause. I've tried reinstalling both foobar and the streamdeck extension multiple times from multiple sources. Any help would be greatly appreciated.

**Comment by MrGamy** [comment]
Same Issue here. I suspect one of the latest Stream Deck Updates. I am currently on Version 5.3.1
Let me know if I can help locate the issue

**Comment by SolidplasmaStudios** [comment]
Any updates on this? @MrGamy Did you find a fix?

**Comment by MrGamy** [comment]
nope, nothing unfortunately :/ the standard "media start/stop/pause" buttons from streamdeck work okayish right now.
Waiting on a response from davidborzek or @9ballsyndrome since they're the ones last working on this.

**Comment by MrGamy** [comment]
@SolidplasmaStudios it seems to be working again. Guess Elgato fixed something in the streamdeck software that broke the plugin

**Comment by nhustak** [comment]
Stopped working for me with the latest stream deck update 8(
